test/issue1867-lowmem.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+. $(dirname "${BASH_SOURCE[0]}")/common.sh
+DIR=$(dirname "${BASH_SOURCE[0]}")
+
+if ! { ${DUB} build --root ${DIR}/issue1867-lowmem -v -f 2>&1 || true; } | grep -cF " -lowmem " > /dev/null; then
+    die $LINENO 'DUB build with lowmem did not find -lowmem option.'
```

Comment:
This is not sufficient because of https://issues.dlang.org/show_bug.cgi?id=20699.

Reply:
Thank you for the information !
This PR of passing the `-lowmem` from dub to dmd and the Issue20699 that the `-lowmem` in the passed response file is not actually used seems to be two different things.
So, I choose not to do anything to this point.
And also, it seems to me that Issue20699 is a dmd issue, but if it has to be fixed in the dub, a separate Issue should be created. Well, I'm hoping that the dmd improvements will solve this problem.

Reply:
Sure, I just meant to let you know that this won't work with DMD.